manifests/kubeedge/input-nodeExporterMetrics.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterInput
+metadata:
+  name: node-exporter-metrics
+  labels:
+    fluentbit.fluent.io/enabled: "true"
+    fluentbit.fluent.io/component: logging
+spec:
+  nodeExporterMetrics:
+    tag: kube.*
+    scrapeInterval: 10s
+    path :
```

Comment:
To collect edge node metrics from within a pod, you'll need to mount host /proc/ and /sys/ into the pod.
Please take a look at https://github.com/fluent/fluent-operator/blob/master/docs/plugins/fluentbit/clusterinput/node-exporter-metrics.md and test again

should be changed to 
```yaml
      path:
        procfs: /host/proc
        sysfs: /host/sys
```